cognite/client/data_classes/model_hosting/models.py

```diff
@@ -57,13 +54,12 @@ class ModelVersion(CogniteResource):
     """A representation of a Model version in the model hosting environment.
 
     Args:
-        id (int): Id of the model version.
+        name (str): Name of the model version.
```

Comment:
`version_name` maybe?

Reply:
Agreed. So I think we should have `name` only when there is a single name in the method.

When there are more than one name i a method, I think it makes sence to keep a consistent order throughout. I suggest `model_name` before `version_name` and the same for schedules.

Reply:
Hmm, ok.